# Inteligência de Mercado

## O que é inteligência de mercado?

Inteligência de mercado é o processo sistemático de coleta, análise e interpretação de dados relevantes ao ambiente de negócios, com o objetivo de embasar decisões estratégicas. Mais do que simplesmente observar o mercado, trata-se da formalização da intuição — ou seja, transformar percepções subjetivas e experiências empíricas em um método estruturado e rigoroso. Ao substituir o improviso e o achismo por análises consistentes, a inteligência de mercado permite que as decisões sejam tomadas com base em evidências concretas, reduzindo riscos e aumentando a assertividade.

Este projeto apresenta os primeiros passos de um protótipo de sistema de inteligência de mercado, com foco nas perguntas estratégicas e nas fontes de informação. Trata-se de um sistema dinâmico, que evolui com o tempo, à medida que a equipe e a empresa ganham experiência e ajustam o processo com base no uso e no feedback interno.

## Dados Populacionais
### Qual é a dinâmica populacional de Aracaju?

A tabela abaixo mostra as populações do Brasil, Nordeste, Sergipe e Aracaju nos anos de 2000, 2010 e 2022.

In [106]:
import pandas as pd
dfPop = pd.DataFrame(
    {
        "Populacao": ["Brasil", "Nordeste", "Sergipe", "Aracaju"],
        "2000": [169590693, 47693253, 1781714, 460898],
        "2010": [190732694, 53081950, 2068017, 571149],
        "2022": [203080756, 54658515, 2210004, 602757],
    }
)
dfPop

Populacao       2000       2010       2022
0    Brasil  169590693  190732694  203080756
1  Nordeste   47693253   53081950   54658515
2   Sergipe    1781714    2068017    2210004
3   Aracaju     460898     571149     602757

Vamos incluir na tabela a variação em cada período, para cada região:

In [107]:
dfPop["variação 2000 para 2010"] = (dfPop["2010"] / dfPop["2000"] - 1) * 100
dfPop["variação 2010 para 2022"] = (dfPop["2022"] / dfPop["2010"] - 1) * 100
dfPop

Populacao       2000       2010       2022  variação 2000 para 2010  \
0    Brasil  169590693  190732694  203080756                12.466487   
1  Nordeste   47693253   53081950   54658515                11.298657   
2   Sergipe    1781714    2068017    2210004                16.068965   
3   Aracaju     460898     571149     602757                23.920911   

   variação 2010 para 2022  
0                 6.474014  
1                 2.970059  
2                 6.865853  
3                 5.534108

Observe que Aracaju foi a região destacada que teve o maior crescimento populacional (24%) na década de 2000 para 2010, e teve também um crescimento destacado (5.5%) no último período, acima do crescimento da região Nordeste.

Mas só esses números não bastam para entender a dinâmica populacional de Aracaju. Nós precisamos transformá-los em informação.

###### Qual a origem dos novos moradores de Aracaju? Quantos vêm de outros estados? Quantos se mudam para trabalhar ou estudar?

###### Quais bairros de Aracaju mais cresceram em população nos últimos anos?

###### Como os aracajuanos percebem os diferentes bairros? Quais são vistos como bons para se viver?

###### Como tem evoluído a população nas cidades da região metropolitana de Aracaju?

Essas são perguntas relevantes para o setor imobiliário, e hoje já é possível respondê-las com o apoio de diferentes fontes — desde pesquisas tradicionais e dados governamentais até informações disponíveis na web.

## Dados da Economia de Aracaju

Sergipe fechou o mês de abril de 2025 com um saldo de 2849 novas vagas de empregos formais. Em Aracaju, foram registradas 7.532 admissões e 5.893 demissões, resultando em um saldo positivo de 1.639 novos vínculos formais, o que corresponde a aproximadamente 57% do saldo positivo de Sergipe. As vagas geradas em Aracaju representam um aumento de aproximadamente 60% em relação ao mesmo período de 2024. Os três setores que mais geraram emprego foram o setor de Serviços (868), Construção Civil (368) e Comércio (247).

##### Em que etapa do ciclo econômico encontra-se o mercado imobiliário em Aracaju?

 O que os números acima nos dizem sobre a economia local de Aracaju? Esses números nos ajudam a identificar em qual ponto do ciclo econômico a economia de Aracaju encontra-se? Quais os hábitos de consumo das pessoas nessa etapa do ciclo econômico?
 Quantas empresas abriram e quantas fecharam em Aracaju no mesmo período em cada setor da economia?

##### Esse movimento indica um aquecimento na economia?

E sobre as pessoas que não conseguem comprovar renda formal, qual fatia do mercado elas representam? Essas pessoas tem acesso ao crédito bancário?

##### Existe algum programa para tentar trazer essas pessoas para dentro do sistema bancário e conseguir ter aceso ao crédito imobiliário?

Mais uma vez, essas são questões fundamentais para todo o mercado imobiliário. Neste estágio, nosso sistema de inteligência de mercado pode se apoiar em dados de diferentes áreas da empresa, como os setores econômico, comercial e de mercado, entre outros.

## CENSO 2022 IBGE

 Em 2022 o IBGE realizou o censo mais detalhado da história do Brasil. Infelizmente esses dados ainda não estão todos disponíveis.
 Para exemplificar as nossas análises, nós usaremos dados do último censo divulgado pelo IBGE, o censo de 2010.
 É importante lembrar que o salário mínimo à época era 510 reais.
 Nós não consideraremos todos os dados disponíveis no IBGE, apenas algumas classes de dados que nos permitem
 desenvolver uma análise preliminar.

## Qual a distribuição de imóveis entre casas e apartamentos em Aracaju?

##### E qual a situação de cada imóvel com relação ao financiamento bancário?

In [110]:
dfDomisAju = pd.DataFrame(
    {
        "Domicilios": ["Casa", "Apartamento"],
        "Total": [119280, 37902],
        "Proprio": [87827, 26812],
        "Quitado": [83167, 17204],
        "Nao-Quitado": [4659, 9608],
        "Alugado": [24889, 9786],
    }
)
dfDomisAju

Domicilios   Total  Proprio  Quitado  Nao-Quitado  Alugado
0         Casa  119280    87827    83167         4659    24889
1  Apartamento   37902    26812    17204         9608     9786

Da tabela acima temos que

In [109]:
dfDomisAju['Total'].sum(), dfDomisAju['Quitado'].sum(), dfDomisAju['Alugado'].sum()

(157182, 100371, 34675)

de um total de 157182 imóveis analisados, 100371 estão quitados e 34675 são alugados.

Partindo de um cenário onde a análise economica é como discutida na seção anterior, podemos nos perguntar:

##### Quem está indo ao mercado em busca de imóveis, está interessado em um imóvel para morar ou para investir?

É claro que sempre há os dois grupos no mercado, porém ao identificá-los nós obtemos uma vantagem competitiva muito grande. Por exemplo, nós podemos fazer a campanha de marketing certa para cada grupo de cliente.

Para melhor desenhar as nossas estratégias é muito importante entender as características dos imóveis disponíveis no mercado.

## Quantos quartos por apartamento em Aracaju?

In [111]:
dfAptQuartosAlugadosAju = pd.DataFrame(
    {
        "Apartamento": ["1-Quarto", "2-Quartos", "3-Quartos"],
        "Quantidade": [2387, 3894, 2956],
    }
)
dfAptQuartosAlugadosAju

Apartamento  Quantidade
0    1-Quarto        2387
1   2-Quartos        3894
2   3-Quartos        2956

A maior parte dos apartamentos em Aracaju tem 2 quartos, seguida por unidades com 3 e 1 quarto. De acordo com o IBGE, a média é de 2,8 moradores por apartamento. Já os imóveis com 4 ou mais quartos eram muito raros na amostra de 2010 e, por isso, foram desconsiderados nesta análise.

#### Quanto custa alugar apartamento em Aracaju (em 2010)?

In [112]:
dfAptQuartosAlugadosFaixaAju = pd.DataFrame(
    {
        "Aluguel": [
            "1/2 a 1 Salário Mínimo",
            "1 a 2 Salário Mínimo",
            "2 a 3 Salário Mínimo",
        ],
        "1-Quarto": [1607, 657, 22],
        "2-Quartos": [2724, 933, 29],
        "3-Quartos": [1742, 1038, 67],
    }
)
dfAptQuartosAlugadosFaixaAju

Aluguel  1-Quarto  2-Quartos  3-Quartos
0  1/2 a 1 Salário Mínimo      1607       2724       1742
1    1 a 2 Salário Mínimo       657        933       1038
2    2 a 3 Salário Mínimo        22         29         67

Compreender os valores de aluguel em Aracaju é essencial. Apesar do desejo comum pela casa própria, acompanhar a dinâmica dos aluguéis ajuda a orientar melhor nossas estratégias de venda e marketing. Campanhas como “Saia do aluguel!” ou “Compre para investir!” podem ser eficazes, dependendo do cenário atual do mercado.

A seguir vamos continuar com uma análise local focada no bairro Luzia, um dos bairros mais tradicionais capital Sergipana.

## População do bairro Luzia

In [113]:
dfPopLuzia = pd.DataFrame(
    {
        "Moradia": ["Casa", "Apartamento", "Total"],
        "População": [8819, 11293, 20430],
    }
)
dfPopLuzia

Moradia  População
0         Casa       8819
1  Apartamento      11293
2        Total      20430

A maior parte da população do bairro Luzia vive em apartamentos (IBGE 2010).

### O mercado imobiliário está saturado no bairro Luzia?

O bairro Luzia é uma das áreas mais bem localizadas de Aracaju, o que também contribui para sua alta densidade populacional. Afirmar que o bairro está saturado para grandes empreendimentos residenciais não significa que não existam oportunidades de negócios na região. Pelo contrário — com o crescimento populacional, o comércio local tende a crescer junto. Uma das funções da inteligência de mercado é justamente fazer o “teste vocacional” de um terreno: talvez não haja mais espaço para grandes projetos habitacionais, mas pode haver demanda por empreendimentos comerciais, como salões ou lojas de serviço.

## Situação dos imóveis do bairro Luzia

In [114]:
dfDomisLuzia = pd.DataFrame(
    {
        "Domicilios": ["Casa", "Apartamento"],
        "Total": [2441, 3909],
        "Proprio": [1912, 2760],
        "Quitado": [1847, 2200],
        "Nao-Quitado": [65, 560],
        "Alugado": [464, 1045],
    }
)
dfDomisLuzia

Domicilios  Total  Proprio  Quitado  Nao-Quitado  Alugado
0         Casa   2441     1912     1847           65      464
1  Apartamento   3909     2760     2200          560     1045

## Renda média nominal no bairro Luzia

In [115]:
dfRendimentoLuzia = pd.DataFrame(
    {
        "Grupo": [
            "Média Geral",
            "20 a 24 anos",
            "25 a 29 anos",
            "30 a 34 anos",
            "35 a 39 anos",
            "40 a 44 anos",
            "45 a 49 anos",
            "50 a 54 anos",
            "55 a 59 anos",
        ],
        "Rendimento Médio Nominal": [2385,915,1633,2159,2388,2714,3063,3294,3565],
    }
)
dfRendimentoLuzia

Grupo  Rendimento Médio Nominal
0   Média Geral                      2385
1  20 a 24 anos                       915
2  25 a 29 anos                      1633
3  30 a 34 anos                      2159
4  35 a 39 anos                      2388
5  40 a 44 anos                      2714
6  45 a 49 anos                      3063
7  50 a 54 anos                      3294
8  55 a 59 anos                      3565

Observe que quanto maior a idade, maior a renda média. Essa é uma observação interessante mas precisa ser interpretada no devido contexto, que é:

##### O que cada grupo destacado acima busca no mercado imobiliário?

##### É razoável supor que as pessoas que compram um segundo imóvel pertence a um grupo de idade mais avançado?

Esse grupo tende a ter uma renda mais alta e maior probabilidade de já ter um imóvel quitado. Por isso, é fundamental buscar dados que validem nossas hipóteses — entender o perfil de cada público é essencial para definir a melhor forma de comunicação.

## Estoque de imóveis

A seguinte amostra de imóveis do bairro Luzia foi extraída de um site de anúncios da internet.
Vamos focar aqui em imóveis de 300mil a 500mil, aproximadamente, o que representa, por exemplo, a faixa 4 do programa "Minha Casa Minha Vida".

In [116]:
dfEstoqueLuzia = pd.DataFrame(
    {
        "Apartamento": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
        "Novo": [True, True, True, True, True, False, False, False, False, False],
        "Valor de venda": [
            420000,
            310000,
            350000,
            387000,
            461000,
            315000,
            370000,
            530000,
            350000,
            360000,
        ],
        "Área em m2": [63, 56, 55, 60, 74, 82, 86, 89, 68, 98],
        "Quartos": [3, 3, 2, 2, 3, 3, 3, 3, 3, 3],
        "Suites": [1, 1, 1, 1, 1, 1, 1, 1, 1, 2],
        "Banheiros": [1, 1, 1, 1, 1, 1, 1, 2, 1, 2],
        "Vagas de garagem": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
        "Varanda": [True, True, True, True, True, True, True, True, True, True],
    }
)
dfEstoqueLuzia

Apartamento   Novo  Valor de venda  Área em m2  Quartos  Suites  Banheiros  \
0            1   True          420000          63        3       1          1   
1            2   True          310000          56        3       1          1   
2            3   True          350000          55        2       1          1   
3            4   True          387000          60        2       1          1   
4            5   True          461000          74        3       1          1   
5            6  False          315000          82        3       1          1   
6            7  False          370000          86        3       1          1   
7            8  False          530000          89        3       1          2   
8            9  False          350000          68        3       1          1   
9           10  False          360000          98        3       2          2   

   Vagas de garagem  Varanda  
0                 1     True  
1                 1     True  
2                 1     True  
3                 1     True  
4                 1     True  
5                 1     True  
6                 1     True  
7                 1     True  
8                 1     True  
9                 1     True

Vamos incluir na tabela o preço do "metro quadrado" de cada imóvel e calcular o preço médio do m2 dos imóveis novos e usados da nossa amostra:

In [117]:
dfEstoqueLuzia['Valor do m2'] = (dfEstoqueLuzia['Valor de venda']/dfEstoqueLuzia['Área em m2']).astype(int)
dfEstoqueLuzia

Apartamento   Novo  Valor de venda  Área em m2  Quartos  Suites  Banheiros  \
0            1   True          420000          63        3       1          1   
1            2   True          310000          56        3       1          1   
2            3   True          350000          55        2       1          1   
3            4   True          387000          60        2       1          1   
4            5   True          461000          74        3       1          1   
5            6  False          315000          82        3       1          1   
6            7  False          370000          86        3       1          1   
7            8  False          530000          89        3       1          2   
8            9  False          350000          68        3       1          1   
9           10  False          360000          98        3       2          2   

   Vagas de garagem  Varanda  Valor do m2  
0                 1     True         6666  
1                 1     True         5535  
2                 1     True         6363  
3                 1     True         6450  
4                 1     True         6229  
5                 1     True         3841  
6                 1     True         4302  
7                 1     True         5955  
8                 1     True         5147  
9                 1     True         3673

In [118]:
dfEstoqueLuzia[dfEstoqueLuzia['Novo'] == True]['Valor do m2'].mean(), dfEstoqueLuzia[dfEstoqueLuzia['Novo'] == False]['Valor do m2'].mean()

(6248.6, 4583.6)

Portanto, o preço do m2 para imóveis novos na nossa amostra é 6240 reais e para imóveis usados é 4583 reais.

O que essa diferença de

In [119]:
((6248/4583) - 1)*100

36.329914902902026

aproximadamente 36,33% no valor do "m2 novo" para o "m2 usado" representa?
O ponto é que sem inteligência de mercado nós não conseguimos responder.
Mais precisamente nós precisamos de dados para responder as seguintes perguntas:

##### O quanto uma pessoa está disposta a pagar a mais por um imóvel novo "equivalente" a um imóvel usado?

Em termos mais gerais:

##### Sob quais condições nós devemos considerar imóveis usados como sendo estoque em nossas análises?

É importante identificar quando vale mais a pena para o investidor comprar um imóvel usado para aluguel do que investir em lançamentos imobiliários.

## Velocidade de vendas e análise da concorrência

A velocidade de vendas é um fator importantíssimo na análise de viabilidade dos empreendimentos imobiliários. Ninguém quer lançar o que não vai vender.

#### Quantas unidades projetamos vender em cada etapa (lançamento, construção, entrega)?

Nem sempre o produto que vendeu bem vai continuar vendendo. É muito importante aprender com nossos erros e acertos, e muito lucrativo quando aprendemos com os erros e acertos da concorrência. Por isso é muito importante entendemos quais são as demandas atuais do mercado. 

Há muitas ferramentas que podemos utilizar para entender melhor os desejos do mercado consumidor. Uma etapa fundamental é o "pós-venda". Por que conseguimos vender? Também é muito importante entender quando não conseguimos vender.

#### Quem queria já comprou. E agora? Como identificar se ainda há demanda no mercado para um determinado produto?

Muitas vezes, os dados já estão dentro da própria empresa — o que falta é transformá-los em informação útil. Por exemplo, podemos analisar históricos de atendimento para entender qual corretor ou imobiliária se comunica melhor com cada perfil de cliente. Da mesma forma, registros de visitas aos stands de vendas podem revelar o que mais atrai os clientes e quais são suas principais dores. Esses dados ajudam a ajustar as abordagens e melhorar os resultados.

Claro, tudo isso só é possível com uma postura verdadeiramente orientada por dados por parte da empresa.

## Acesso ao crédito imobiliário e análise de sensibilidade das hipóteses

É fundamental entender a disponibilidade de crédito para melhor posicionar o nosso produto no mercado.

#### Quem é o meu público-alvo tem condições de comprar o que estou oferecendo?

Analisar dados não é prever o futuro — é compreender o contexto e explorar possíveis caminhos. É essencial saber como nossos modelos respondem a pequenas mudanças nas hipóteses.

#### Se o cenário econômico mudar, mesmo que levemente, como nosso produto se ajustará no mercado?

Diversos fatores podem influenciar o posicionamento do nosso produto no mercado — e é tanto necessário quanto possível identificá-los e analisá-los.

## Conclusão

O que apresentamos acima representa mais uma proposta de postura a ser adotada pela empresa do que um detalhamento completo de um projeto de inteligência de mercado. Trata-se de um processo contínuo, que começa de forma simples e evolui com o tempo. O que foi mostrado são possibilidades, caminhos que podem ser seguidos conforme os objetivos e a maturidade da empresa. É fundamental que todos os setores participem da construção dessa inteligência, contribuindo com informações e experiências para que o sistema se torne cada vez mais eficiente e estratégico.